lib/AppInfo/Application.php

```diff
@@ -78,4 +81,33 @@ public function __construct(array $urlParams = array()) {
 		});
 	}
 
+	public function boot() {
```

Comment:
So if I understand correctly, this function is called for the login screen  (Is it the login screen only or all the screens?)  and injects the javascript that will change the login message.